cmd/start.go

```diff
@@ -96,7 +105,26 @@ var startCmd = &cobra.Command{
 			<-c
 			logger.Info("stopped message processor")
 			return nil
-		})
+		}
+
+		if embedNats {
+			opts := &server.Options{}
```

Comment:
probably also want to expose some of the server options in case we need to change port or credentials